Update EVE HGEC Eyecandy Variants Expansion

Sabre142 opened the pull request:
Related to #317.

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

* Unify scattered mod entries into one complete entry
 - Move all ESP entries to the AudioVisual - Models & Textures section.

* Capture the locations of all ESPs using a regex
 - EVE HGEC Eyecandy Variants Expansion:
   https://www.nexusmods.com/oblivion/mods/24078/
 - Seamless - OCOv2: https://www.nexusmods.com/oblivion/mods/45859/

* EVE_StockEquipmentReplacer.esp:

* Add messages
 - Use only one EVE_StockEquipmentReplacer ESP.
 - Use MMM version if Martigen's Monster Mod is detected.
 - Use OOO version if Oscuro's Oblivion Overhaul is detected.

* Remove tag
 - Graphics, mod already has it in the ESP's description.

* EVE_KhajiitFix.esp:

* Add incompatibility
 - With Oblivion_Character_Overhaul.
 - Both mods modify the same subrecord and the same texture.

* Add tag suggestions
 - Mod already has Body-F tag in the ESP's description.
   Modifies female body data part in 000223C7.
 - Deactivate, should be deactivated and imported into the
   bashed patch, so that only the R.Body-F change is imported.
 - NoMerge, should not be merged even though it is technically
   mergeable, so that only the R.Body-F change is imported.
 - Remove Body-Size-F tag in LOOT, mod does
   not change female race height or weight.

* Add cleaning data
 - Version 1.0.

* EVE_StockEquipmentReplacer4FCOM.esp:

* Add message
 - Use only one EVE_StockEquipmentReplacer ESP.

* Add tag suggestions
 - Mod already has Graphics tag in the ESP's description.
 - Deactivate, should be deactivated and imported into the
   bashed patch, so that only the Graphics changes are imported.
 - Not including tags for Names and Stats. This would cause issues
   for users who use updated versions of OOO (v1.5+), while this mod
   was designed for OOO v1.33, and these values have changed between
   OOO versions.

* Add cleaning data
 - Version 1.4.

* EVE_StockEquipmentReplacer4MMM.esp:

* Add requirement
 - Missing Martigen's Monster Mod master.

* Add message
 - Use only one EVE_StockEquipmentReplacer ESP.

* Add tag suggestions
 - Mod already has Graphics tag in the ESP's description.
 - Deactivate, should be deactivated and imported into the
   bashed patch, so that only the Graphics changes are imported.

* Add cleaning data
 - 12 May 2009 (no version).

* EVE_StockEquipmentReplacer for OOO.esp:

* Add tag suggestions
 - Mod already has Graphics tag in the ESP's description.
 - Deactivate, should be deactivated and imported into the
   bashed patch, so that only the Graphics changes are imported.
 - Not including tags for Names and Stats. This would cause issues
   for users who use updated versions of OOO (v1.5+), while this mod
   was designed for OOO v1.33, and these values have changed between
   OOO versions.

* Add cleaning data
 - Version 1.3.

* Armamentium female.esp:

* Add tag suggestions
 - Mod already has Graphics and NoMerge tags in the ESP's description.
 - Deactivate, should be deactivated and imported into the
   bashed patch, so that only the Graphics changes are imported.
 - Not including tag for Stats changes, probably from an old version of
   Armamentarium, and is not the focus of the mod.

* Add cleaning data
 - 12 May 2009 (no version).

* ArtifactsFemaleArmor.esp:

* Add tag suggestion
 - Deactivate, should be deactivated and imported into the
   bashed patch, so that only the Graphics changes are imported.

* Add cleaning data
 - 29 Dec 2009 (no version).

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
+  - name: 'EVE_StockEquipmentReplacer.esp'
+    inc:
+      - 'Mart''s Monster Mod.esm'
+      - 'Oscuro''s_Oblivion_Overhaul.esm'
```
These can be removed as they're redundant with the `*useVersion` messages & are also not in agreement with them as incompatible messages are errors while the `*useVersion` messages you added are warnings.

```suggestion
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
+      - 'Mart''s Monster Mod.esm'
+      - 'Oscuro''s_Oblivion_Overhaul.esm'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'EVE_StockEquipmentReplacer ESP' ]
+        condition: 'many("EVE_StockEquipmentReplacer(4FCOM|4MMM| for OOO)?\.esp")'
```
These `*useOnlyOneX` messages can be moved to a regex entry, namely `EVE_StockEquipmentReplacer(4FCOM|4MMM| for OOO)?\.esp`. Note that you'd also be able to move the `Deactivate` & `NoMerge` tags for all of these plugins to this entry.

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
+        condition: 'many("EVE_StockEquipmentReplacer(4FCOM|4MMM| for OOO)?\.esp")'
+      - <<: *useVersion
+        type: warn
+        subs: [Martigen''s Monster Mod]
```
This string should be wrapped in single-quotes.

```suggestion
        subs: [ 'Martigen''s Monster Mod' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
+        condition: 'active("Mart''s Monster Mod.esm") and not file("EVE_StockEquipmentReplacer4MMM.esp")'
+      - <<: *useVersion
+        type: warn
+        subs: [Oscuro''s Oblivion Overhaul]
```
This message can be refactored using an existing anchor (it includes the `warn` `type` declaration).

```suggestion
      - <<: *useVersionOOO
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
+      - Stats
+    clean:
+      - crc: 0x02EC9CA3
+        util: 'TES4Edit v4.0.4'
```
The cleaning data for `EVE_StockEquipmentReplacer.esp` is the same for xEdit v4.0.4b.

```suggestion
    clean:
      - crc: 0x02EC9CA3
        util: 'TES4Edit v4.0.4b'
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
       - NoMerge
     clean:
       - crc: 0xA28C2C54
         util: 'TES4Edit v4.0.4'
```
```suggestion
    clean:
      - crc: 0xA28C2C54
        util: 'TES4Edit v4.0.4b'
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -3183,42 +3183,111 @@ plugins:
+        itm: 1
+  - name: 'Armamentium female.esp'
+    tag:
+      - Deactivate
```
```suggestion
    tag: [ Deactivate ]
```

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

Co-authored-by: sibir <ine.sibir@gmail.com>

Sabre142 committed:
Update EVE HGEC Eyecandy Variants Expansion

* Moved the *useOnlyOneX message to a regex entry. Also moved the
  Deactivate & NoMerge tags for all of these plugins to this entry.

* Disable *useVersion messages if one
  is using MMM and OOO at the same time.

* Add *useVersion messages to the regex.

sibir-ine commented:
Thanks!
